Source/com/drew/imaging/quicktime/QuickTimeTypeChecker.java

```diff
@@ -36,52 +36,51 @@
         // http://www.ftyps.com
```

Comment:
Please bring changes from drew's code and test if it breaks anything. We ignore this change in the assumption that, this caused exif read failure. 

Please cross check.

Reply:
Updated with drew's code and tested. It is working fine with drew's code.